### ISSUE TITLE
Kronecker Graph generator

### ISSUE THREAD

**Issue by jpfairbanks** [rohitvarkey/StingerGraphs.jl#11]
Here is an octave implementation that should be easy to port. 
https://gitorious.org/graph500/graph500?p=graph500:graph500.git;a=blob;f=octave/kronecker_generator.m;h=064b6d4f42194cc95ec4ab375dcbee7c3243b233;hb=01a32a9c16994c3cf30bbf4caabf9be128379416

**Comment by jpfairbanks** [comment]
LICENSE

```
 31 Included Kronecker graph generator:
  32 /* Copyright (C) 2009-2010 The Trustees of Indiana University.             */
  33 /*                                                                         */
  34 /* Use, modification and distribution is subject to the Boost Software     */
  35 /* License, Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at */
  36 /* http://www.boost.org/LICENSE_1_0.txt)                                   */
```

**Comment by rohitvarkey** [comment]
Closed in #12
